basic/exercise5/script5.js

```diff
@@ -1 +1,12 @@
 'use strict';
+
+function firstSunday (yearX, yearY){
+    for (var year = yearX; year <= yearY;  year++) {
+        var d = new Date(year, 0, 1);
+         if (d.getDay() === 0){
+           console.log(`The first day of ${year} is a Sunday.`);
```

Comment:
Please return only year here.
This is what function should do.
What will happen with this result is not what this function should do.